src/services/iotHubManagerService.js

```diff
@@ -46,4 +48,12 @@ export class IoTHubManagerService {
     return HttpClient.delete(`${ENDPOINT}devices/${id}`)
       .map(() => ({ deletedDeviceId: id }));
   }
+
+    /** Returns the account's device group filters */
+    static getDeviceProperties() {
+      return Observable.forkJoin(
+      HttpClient.get(`${ENDPOINT}deviceProperties`).map(toDevicePropertiesModel),
```

Comment:
Did you test this code? `toDevicePropertiesModel` model returns an object but you are using the spread operator into an array, which doesn't make sense and seems like it should cause a type error. The ajax calls don't return arrays so you want to merge the internal objects `{ reported, tags }`. I think this should do what you want (note `import update from 'immutability-helper';`):
```
return Observable.merge(
  HttpClient.get(`${ENDPOINT}deviceProperties`)
  HttpClient.get(`${Config.serviceUrls.deviceSimulation}deviceProperties`)
)
.map(toDevicePropertiesModel)
.reduce(
  (acc, props) => update(acc, {
    reported: { $push: props.reported },
    tags: { $push: props.tags }
  }), 
  { reported: [], tags: [] }
);
```